pkg/input/container/docker.go

```diff
@@ -26,7 +27,7 @@ import (
 )
 
 const defaultSleepDuration = 1 * time.Second
-const DateLayout = "2006-01-02T15:04:05.000000000Z"
+const tagsUpdatePeriod = 10 * time.Second
```

Comment:
Checking the container tags and running a deep equals every 10 seconds feels a bit costly to me, because I wouldn't expect docker tags to change _that_ often. Shouldn't we raise the period here ?

Reply:
Good question, it's a tradeoff here. I think docker tags _almost never_ change, but as a user if my tags changed I would like to see it updated within a few seconds, thus the frequency.

I don't think it's that costly, tags is a dict of ~4 strings with less than 20 chars so it's pretty fast. Also, it's in a different goroutine, so it won't slow down the agent, just use a bit more CPU from time to time.

I suggest we leave as is, and if we see it being a problem we'll move to 60s and make it configurable so users can override it if needed